docs/overview.md

```diff
@@ -0,0 +1,98 @@
+
+Octopus Cloud is the hosted version of Octopus Deploy. We designed Octopus Cloud and self-hosted Octopus to provide the same functionality; however, there are some minor differences, for instance, with Octopus Cloud, we're [responsible](/docs/administration/security/index.md#responsibility) for taking backups, upgrading the service, and maintaining and monitoring the underlying systems.
+
+Learn more about [Octopus Cloud](/docs/octopus-cloud/index.md).
+
+### The Octopus Web Portal
+
+Whether you're self-hosting the Octopus Server, or using Octopus Cloud, the Octopus Web Portal is where you'll manage your infrastructure, projects, access the built-in repository, grant your team access to projects, and create your automated deployments.
+
+## Infrastructure
+
+Octopus Deploy organizes your deployment targets (the machines and services you deploy software to) into groups called environments. Typical environments are **Development**, **Test**, and **Production**.
+
+With Octopus Deploy your deployment targets could be Windows servers, Linux servers, Microsoft Azure, AWS, Kubernetes Clusters, Cloud Regions, or even an Offline Package Drop.
+
+Organizing your infrastructure into environments lets you define your deployment processes (no matter how many steps or deployment targets are involved) and have Octopus deploy the right versions of your software, with the right configuration, to the right environments at the right time.
+
+Learn more about managing your [infrastructure](/docs/infrastructure/index.md).
+
+## Packaging Applications
+
+Before you can deploy software with Octopus Deploy, you need to bundle all the files required for the software to run into a supported package. The package must be versioned and stored in a repository. Octopus Deploy includes a built-in repository, but you can also host your packages in external repositories. We recommend configuring your existing tool chain to push packages automatically to the repository. Octopus supports a variety of package formats.
+
+Learn more about [packaging your applications](/docs/packaging-applications/index.md) or how to automate your existing tool chain to push packages to your Octopus Deploy server with our [Build Server Integrations](/docs/octopus-rest-api/index.md).
+
+## The Deployment Process
+
+Octopus Deploy is designed to work with teams following agile software development methodologies, that is, continuously deploying software, iterating, making changes, and redeploying.
+
+Before you can deploy your software, you need to create a project which will include your deployment process and all the information needed for your teams to successfully redeploy every time. The deployment process is a series of steps, each step contains a specific action (or set of actions) that is executed as part of the deployment process every time your software is deployed. Octopus Deploy includes a range of built in steps you can use in your deployment processes.
```

Comment:
"successfully redeploy every time" should be "successfully deploy every time" I think. We shouldn't be confusing users on `deploy` vs `redeploy` terminology. Redeploying to me is really re-deploying the same release to the same environment, I.e. when you've needed to update a variable but not bump the release/package version. It's not something we encourage but it's something available to you because sometimes you just need to update some config and re-deploy the same release.

Reply:
Thanks. Good catch!